src/main/scala/strawman/collection/Seq.scala

```diff
@@ -662,6 +664,18 @@ trait LinearSeqOps[+A, +CC[X] <: LinearSeq[X], +C <: LinearSeq[A]] extends Any w
     def next() = { val r = current.head; current = current.tail; r }
   }
 
+  override def size: Int = {
+    var these = coll
+    var len = 0
+    while (!these.isEmpty) {
+      len += 1
+      these = these.tail
+    }
+    len
+  }
+
+  override def length: Int = size
```

Comment:
We should have a policy about which method overrides which so we don't get in tangles.  In any case, when you have two synonymous methods, one should be final.  So in this case, `final def length: Int = size`, though it's a bit odd that the normal `size` defers to `iterator.length`.  It would seem more natural to me to have `length` contain the implementation here.